Code/skyrim_ui/src/assets/i18n/fr.json

```diff
@@ -0,0 +1,155 @@
+      "MESSAGE_TOO_LONG": "Vous ne pouvez pas envoyer un message d'une longueur supérieure à  {{chatMessageLengthLimit}} caractères."
+    },
+    "CONNECT": {
+      "INFO": {
+        "SERVER_ADDRESS": "Connectez-vous à un serveur en utilisant son adresse complète. Utilisez le format  <strong>addresse:port</strong>. Le port par défaut est : <strong>10578</strong>.",
+        "HELGEN": "Ne vous connectez PAS à un serveur si vous êtes toujours dans la séquence d'introduction de Helgen. Assurez-vous d'abord d'avoir fui Helgen."
+      },
+      "ADDRESS": "Addresse",
+      "PASSWORD": "Mot de passe",
+      "SAVE_PASSWORD": "Enregistrer le mot de passe ?",
+      "CONNECT": "Connexion",
+      "CANCEL": "Annuler",
+      "PUBLIC_SERVERS": "Serveurs publics",
+      "ERROR": {
+        "CONNECTION": "Impossible de se connecter au serveur spécifié. Veuillez vous assurer que vous avez entré l'adresse correcte et que vous ne rencontrez pas de problèmes de réseau.",
+        "VERSION_MISMATCH": "Vous ne pouvez pas vous connecter au serveur car il ne dispose pas de la même version de Skyrim Together que vous.",
+        "INVALID_ADDRESS": "Le format de l'adresse est invalide. (Adresse:Port)"
+      }
+    },
+    "DISCONNECT": {
+      "PROMPT": "Vous êtes sûr de vouloir vous déconnecter ?",
+      "PROCEED": "Allez-y",
+      "CANCEL": "Annuler"
+    },
+    "ERROR": {
+      "OK": "Ok"
+    },
+    "NOTIFICATIONS": {
+      "MESSAGE_FROM": "Message de"
+    },
+    "PARTY_MENU": {
+      "ACCEPT_INVITE": "Accepter l'invitation de {{name}}",
+      "LAUNCH_PARTY": "Lancer la partie",
+      "INVITES": "Invités",
+      "NO_INVITES": "Vous n'avez pas d'invitations",
+      "EXPERIENCE_WARNING": "Pour une expérience de quête optimale, veillez à lire le guide de jeu sur notre wiki, que vous pouvez trouver via notre site web, <b>skyrim-together.com</b>. Voici un petit résumé des directives :",
+      "GUIDELINES": {
+        "PARTY": "En créant un groupe, vous devenez en quelque sorte l'hôte, autrement dit le chef de groupe.",
+        "PARTY_LEADER_TALK": "Seul le chef du groupe peut parler aux PNJ des quêtes pour progresser dans ces dernières.",
+        "PARTY_LEADER_LOOT_QUESTITEMS": "Seul le chef de groupe peut récupérer les objets de quête.",
+        "KEEP_PARTY_LEADER": "Tenez-vous-en à un seul chef de groupe par partie.",
+        "ONE_PARTY_PER_SERVER": "Tenez-vous-en à une partie par serveur."
+      },
+      "NO_PLAYERS_IN_PARTY": "Il n'y a encore personne dans votre groupe. Invitez-les via la liste des joueurs !",
+      "TABLE_HEADERS": {
+        "LEVEL": "Niv.",
+        "NAME": "Nom",
+        "LOCATION": "Localisation"
+      },
+      "ACTIONS": {
+        "KICK": "Exclure",
+        "TELEPORT": "Téléporter",
+        "MAKE_LEADER": "En faire le chef de groupe"
+      },
+      "LEAVE_PARTY": "Quitter le groupe"
+    },
+    "PARTY_LIST": {
+      "NO_OTHER_PLAYERS": "Il n'y a personne ici à part nous poulets ",
```

Comment:
Not sure if its the joke is using "nous" as "us", to describe chickens or if its a misspelling, could probably do without the joke entirely tbh